crates/biome_cli/src/commands/mod.rs

```diff
@@ -183,6 +183,10 @@ pub enum BiomeCommand {
         #[bpaf(long("write"), switch)]
         write: bool,
 
+        /// Suppress existing diagnostics with a `// biome-ignore` comment (Only implemented for JavaScript)
```

Comment:
```suggestion
        /// Fix diagnostics with suppression comments if the language supports it. 
```

I figured we could use a more broad phrasing, without citing JS